Close every file after processing, not at the end of program.
Index: html2text/html2text.C
===================================================================
--- html2text.orig/html2text.C
+++ html2text/html2text.C
@@ -376,6 +376,7 @@ main(int argc, char **argv)
     );
 
     if (parser.yyparse() != 0) exit(1);
+	uis.close();
   }
 
   return 0;
